l3_dump.py

```diff
@@ -218,6 +210,69 @@ def parse_string_offsets(input_str:str) -> (dict,int):
     # Return # of valid-lines-parsed count, so pytests can verify it.
     return (_strings, nlines)
 
+# #############################################################################
+def l3_unpack_loghdr(file_hdl) -> (int, int):
+    """
+    Unpack the header struct of a L3-log file and identify key fields.
+    We are unpacking a struct laid out like the following:
+
+    typedef struct l3_log
+    {
+        uint64_t        idx;
+        uint64_t        fbase_addr;
+        uint32_t        pad0;
```

Comment:
rather than padding, should use `__attribute__((align(n)))` ?

Reply:
I'd like to stay with the scheme of externalizing pad-bytes added for compiler-padding as, then, it's apparent that some spare bytes are available for future development.

Also, I'm a bit leery of relying on compiler __attribute__s -- just in case those are not supported similarly on all compilers (e.g. clang, MS-Visual C++ etc.)